library/dash/mdhd_contents.cc

```diff
@@ -73,7 +73,7 @@ size_t MdhdContents::Parse(const uint8_t* buffer, size_t length) {
     modification_time_ = ntohllFromBuffer(ptr);
     ptr += sizeof(uint64_t);
     timescale_ = ntohllFromBuffer(ptr);
```

Comment:
Thanks for your reply. I updated and pushed it again.  I am also wondering if there could be a release from project(cdm_player_ios) which is using this libs for this fixing.